loky/backend/fork_exec.py

```diff
@@ -35,9 +35,12 @@ def close_fds(keep_fds):  # pragma: no cover
 
 def fork_exec(cmd, keep_fds):
 
+    # copy the environment variables to set in the child process
+    child_env = os.environ.copy()
```

Comment:
What about making it possible to pass environment variables to override some variables only in the children without changing the `os.environ` dict of the main process?

This new `env` dict could be passed as an argument to the `get_reusable_executor` function.

Reply:
That was what I did before but I am rather hesitant as it means changing the API for `Process` compared to the one from `multiprocessing`. As we do not implement this change for other `start_method`, it would mean having a different way of starting process depending on the context.